.github/ISSUE_TEMPLATE/release.md

```diff
@@ -8,61 +8,52 @@ title: Manta {{ SET_VERSION }} Release checklist
-
-Note: Usually update client first then runtime.
-
-### Client Releases
-
-- [ ] Verify that each crate's `version` has been bumped from previous release.
-- [ ] Update client of Baikal relay nodes.
-- [ ] Update client of Calamari-Testnet @ Baikal nodes.
-- [ ] Update client of Calamari-Testnet @ Moonbase-Relay nodes.
-- [ ] Update client of Dolphin @ Baikal nodes.
-
-### All Releases
-
-- [ ] Check that a draft release has been created at
-    https://github.com/Manta-Network/Manta/releases with relevant [release
-    notes](#release-notes)
-- [ ] Check that build artifacts have been added to the
-    draft-release
-- [ ] Coordinate with marketing team for documentation updates and other relevant tasks.
-- [ ] Update changelog.
 - [ ] If the release contains any changes that break/change functionality used in https://github.com/Manta-Network/sdk (e.g. RPC changes, see also [extrinsic ordering](#extrinsic-ordering)), raise a PR there and **block this release** until your PR has been merged and incorporated in a new SDK release.
+
+# Deploy to public testnet
+    - [ ] execute client upgrade on calamari @ moonbase-relay nodes if needed
+    - [ ] execute runtime upgrade to calamari @ moonbase-relay and verify network stability.
+    - [ ] execute client upgrade on calamari @ kusama-internal nodes if needed
+    - [ ] execute runtime upgrade to calamari @ kusama-internal and verify network stability.
+- [ ] check network health metrics like average block times, block authors, etc with this tool https://parachain-utilities.vercel.app/
+- [ ] Coordinate with marketing team for documentation updates and other relevant tasks.
+- [ ] Monitor [Grafana Node Explorer](https://grafana.pulse.pelagos.systems/d/rYdddlPWk/node-exporter-full) for anomalies in our nodes' memory, cpu, disk and network usage. These would include but are not limited to: memory leaks, cpu spikes, spike in tcp sockets waiting to close, etc. Make sure to take a look at all of the available graphs because some problems might only be visible in views that are collapsed by default
 - [ ] Check that the new client and/or runtime versions have [burned-in](#burn-in) without issue for at least 3 days.
-- [ ] Before declaring a successful burn-in make sure to check for anomalies in our nodes' memory, cpu, disk and network usage via the [Grafana Node Explorer](https://grafana.pulse.pelagos.systems/d/rYdddlPWk/node-exporter-full). These would include but are not limited to: memory leaks, cpu spikes, spike in tcp sockets waiting to close, etc. Make sure to take a look at all of the available graphs, because some problems might only show up in the collapsed views.
 - [ ] Keep an eye out on the [manta status dashboard](https://status.manta.network/) for additional metrics like outages.
 
-Note: Do not publish draft releases from PR branches, because those branches will be deleted when the PR is merged.
```

Comment:
why is this note removed

Reply:
draft releases only get created on manta branch, we changed the CI

Reply:
Can't it be triggered also from a tag (which you can do from the dev branch), there are 3 ORs - [qa9QA7HC6SR82Tbsq2PSn9xQ3sG56Q3ToUENZt5bLvRJ7Rf](https://github.com/Manta-Network/Manta/blob/manta/.github/workflows/publish_draft_releases.yml#L36)
That's the main thing this line is avoiding.

Reply:
I think the real fix here is to only run this workflow when it's run from a tag that starts with `v`
Now that this doesn't run any tests, there's no real reason to run it as part of release branches or on manta branch anyway.

I fixed it by getting rid of the ifs and explaining the process